content/md/imports-exports/tailored-import.md

```diff
@@ -138,6 +138,10 @@ You can use operations to transform the data in order to make the import process
 New operations will come along the year to help you save time importing your product information. Do not hesitate to contact your Customer Success Manager if you want to help us by communicating your use case & needs.
 :::
 
+:::warning
+Adding the operations in the correct order is essential. For instance, if you want to convert ` this text ` to `This text`, you need to add the operations `Remove whitespaces at the beginning and the end`, then `First letter uppercase`, and not the contrary.
```

Comment:
IMO, for these two examples the order does not really matter, it is much more important when you want to split then replace for instance 🤔

Reply:
Updated paragraph